deploy/caddy-config/Caddyfile

```diff
@@ -42,6 +42,36 @@
 			}
 		}
 	}
+	
+	@jupyter_websocket {
+		header Connection *Upgrade*
+		header Upgrade websocket
+	}
+
+	handle @jupyter_websocket {
+		reverse_proxy https://{$PROJECT_NAME}-trex:33000 {
```

Comment:
websocket connections will first be validated in trex before being forwarded to jupyter